rules/rules.js

```diff
@@ -334,6 +336,7 @@ const getTriggeredData = function (input) {
                 d.eventType = "time";
                 d.triggerType = "GenericCronTrigger";
                 d.triggerTypeOld = "TimerTrigger";
+                d.module = "CRON";
```

Comment:
Whats the purpose of injecting this here?

Reply:
for CRON type of triggers, the input is empty.  It would be also totally acceptable to have d.module an empty string.